recipe/meta.yaml

```
@@ -29,6 +29,9 @@ requirements:
     - liblapack
   run:
     - python
+  run_constrained:
+    # enforce eviction of package from anaconda defaults
+    - numpy-base 9999
```

This is more restrictive as `0a0` should be near to the minimal version conda can digest.
```suggestion
    - numpy-base <0a0
```

---

Numpy 9999 is pretty unrealistic already, no? 😅 
And it's in line with other feedstocks, e.g. [pypy](https://github.com/conda-forge/pypy3.6-feedstock/blob/master/recipe/meta.yaml#L67-L69) resp. what @isuruf suggested in the title of #160

That said, I don't care much either way whether it's 9999, 99999999999, or <0a0...